codec-redis/src/main/java/io/netty/handler/codec/redis/PushRedisMessage.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.internal.UnstableApi;
+
+import java.util.Collections;
+import java.util.List;
+
+@UnstableApi
+public class PushRedisMessage extends AbstractCollectionRedisMessage {
```

Comment:
final

Reply:
I made `final` class but it caused build failure, because check-style about `RedundantModifier` of `children()` method.
So, I need remove redundant modifier `final` of `children()` method?